oaknational/Oak-Web-Application
# feat: update footer and menu with eyfs - LESQ-490

## Description

-Adds links to footer and hamburger for eyfs

Fixes #

## How to test

1. Go to https://deploy-preview-2239--oak-web-application.netlify.thenational.academy
Open hamburger - you should see a internal link to eyfs
Scroll down to footer - you should see a internal link to eyfs
